Extensions/hideavatars.js

```diff
@@ -0,0 +1,230 @@
+//* TITLE Hide Avatars **//
+//* VERSION 0.1.0 **//
+//* DESCRIPTION Hides avatars on a per-url basis **//
+//* DEVELOPER dlmarquis **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.hideavatars = new Object({
+
+	running: false,
+	
+	blognames: [],
+	
+	run: function() {
+
+		this.running = true;
+
+		XKit.tools.init_css("hideavatars");
+
+		XKit.extensions.hideavatars.load_blogs();
+		
+		XKit.post_listener.add("hide_avatars", XKit.extensions.hideavatars.hide_avatars);
+		XKit.extensions.hideavatars.hide_avatars();
+		
+	},
+	
+	hide_avatars: function() {
+	    
+	    XKit.extensions.hideavatars.blognames.forEach(function(blogname) {
+	        $(".post_avatar_link").filter(function( index ) {
+	            return $(this).attr("href").slice(7).split(".")[0] == XKit.extensions.hideavatars.blogname.title;
```

Comment:
I could be wrong, but I think `XKit.extensions.hideavatars.blogname.title` should be `blogname.title`.